README.md

```diff
@@ -116,6 +116,17 @@ These instructions will install the development environment into your local mach
 	```
 	This file will be located under ```<PROJECT-ROOT>/paw-webapp/webapp/target```.
 
+## Usage
+
+The application can be used without being authenticated, but the actions that can be performed is limited. For example, you can search for games, but you won't be able to write a review.
```

Comment:
`...but the actions that can be performed ARE limited.`

Reply:
Si, venía escribiendo otra cosa, y dsp cambie a actions.

Reply:
Listo, arreglado